tournamentWinner.js

```diff
@@ -0,0 +1,34 @@
+competitions = [
+    ["HTML", "C#"],
+    ["C#", "Python"],
+    ["Python","HTML"],
+]
+results = [0, 0, 1]
+
+function WhoIstournamentWinner(comp) {
+    var hashMap = new Map();
+    var tounrnWin = "";
+    var winner ="";
```

Comment:
Instead of 4 vars try to use 1 var and one object, and this object will hold all the results.